validator/project_validator.go

```diff
@@ -259,13 +264,13 @@ func CheckYamlStrict(yamlBytes []byte) ValidationErrors {
 // verify that the project configuration semantics and configuration syntax is valid
 func CheckProjectConfigurationIsValid(project *model.Project, pref *model.ProjectRef) error {
 	catcher := grip.NewBasicCatcher()
-	syntaxErrs := CheckProjectSyntax(project, false)
+	syntaxErrs := CheckProjectErrors(project, false)
```

Comment:
similarly, should rename these projectErrs / projectWarnings etc, same with the error strings